pipelines/preparation/preparation_matched_no_extcalibs.yaml

```diff
@@ -0,0 +1,18 @@
+description: Produce matched catalogs
+tasks:
+  matchCatalogsTract:
+    class: lsst.faro.preparation.TractMatchedPreparationTask
+    config:
+      python: |
+        config.doApplyExternalPhotoCalib = False
```

Comment:
Previously, we have used a convention to not set the configuration in the pipeline file when the default values are desired.

Reply:
Good point. Removing those configs would make this file exactly the same as `preparation_matched.yaml`, so I just removed it.